draft-ietf-masque-connect-udp-listen.md

```diff
@@ -213,8 +219,14 @@ COMPRESSION_CLOSE with the context ID (see {{capsulecloseformat}}) if it
 doesn't wish to support compression for the given Context ID (For example, due
 to the memory cost of establishing a list of mappings per target per client).
 If the compression was rejected, the client and proxy will instead use an
-uncompressed context ID (See {{uncompressed}}) to exhange UDP payloads for the
-given target, if those have been enabled.
+uncompressed context ID (See {{uncompressed}}) to exchange UDP payloads for the
+given target, if those have been enabled. Only one Context ID MUST be used per
```

Comment:
This MUST isn't really normative, we already have a MUST below that accomplishes this. So here I would say "Only one Context ID can be used..."